.gitignore

```diff
@@ -1,6 +1,16 @@
 # my stuff
 docs/_build/**
 .vscode/**
+cybergis_compute_user.json
+core.2435
```

Comment:
Small thing: change this to `core.*` to cover any random number that might pop-up.